app/js/views/captain.js

```diff
@@ -12,14 +12,6 @@ define(
                 disabled: true
             },
             {
-                // Unsure if user wants to view last_editor here?
```

Comment:
I found the following in old forms.py, and I assume the property should not be included here. 

class CaptainForm(djangoforms.ModelForm):
	  class Meta:
	    model = models.Captain
	    exclude = ['modified', '**last_editor**', 'last_welcome', 'search_prefixes']

Reply:
Correct, we should omit 'last_editor' from forms.